cocotb/decorators.py

```diff
@@ -567,6 +570,8 @@ async def f(*args, **kwargs):
         self.timeout_unit = timeout_unit
         self.expect_fail = expect_fail
         if expect_error is True:
+            warnings.warn(
+                "`except_error=True` option to `cocotb.test` has been deprecated, specify an expected Python Exception instead")
```

Comment:
```suggestion
                "`expect_error=True` option to `cocotb.test` has been deprecated, specify an expected Python Exception instead")
```
Nobody excepts the Spanish Expection!